README.md

```diff
@@ -2,157 +2,317 @@
 
 ![wizard](https://user-images.githubusercontent.com/8126674/44388234-320aac00-a55a-11e8-879d-5dca68512031.png)
 [![Build status](https://secure.travis-ci.org/kowainik/summoner.svg)](http://travis-ci.org/kowainik/summoner)
-[![MPL-2.0 license](https://img.shields.io/badge/license-MPL--2.0-blue.svg)](https://github.com/kowainik/summoner/blob/master/LICENSE)
 [![Hackage](https://img.shields.io/hackage/v/summoner.svg)](https://hackage.haskell.org/package/summoner)
-[![Stackage LTS](http://stackage.org/package/summoner/badge/lts)](http://stackage.org/lts/package/summoner)
-[![Stackage Nightly](http://stackage.org/package/summoner/badge/nightly)](http://stackage.org/nightly/package/summoner)
-[![Hackage-Deps](https://img.shields.io/hackage-deps/v/summoner.svg)](http://packdeps.haskellers.com/reverse/summoner)
+[![MPL-2.0 license](https://img.shields.io/badge/license-MPL--2.0-blue.svg)](https://github.com/kowainik/summoner/blob/master/LICENSE)
 
 > _So many of our dreams at first seem impossible, then they seem improbable, and
 > then, when we summon the will, they soon become inevitable._
 >
 > Christopher Reeve
 
-Summoner is the tool for creating fully configured production Haskell projects.
+Summoner is a tool for scaffolding fully configured batteries-included production Haskell projects.
+
+Do you want to create a library that is to be uploaded to the Hackage/Stackage, that builds with both Cabal and Stack and supports the latest three major GHC versions?
+Or you are building a production application which uses a custom prelude and has CI with Travis Linux and AppVeyors Windows checks?
+Summoner can help you to do all of that with minimal effort from you - it can even upload the project to GitHub if you want to!
+
+By the way, Summoner operates as either CLI or TUI application, so you can choose what you're more comfortable with and install only the required one.
```

Comment:
Summer is available to be used as either CLI or TUI (we should probably add image or something here or later on that would give a side-by-side view/comparison of both UIs)

and install the UI you require (the one that suits you best)